examples/axisartist/simple_axisline.py

```diff
@@ -37,4 +37,25 @@
 ax.axis["right2"].label.set_text("Label Y2")
 
 ax.plot([-2, 3, 2])
+
+###############################################################################
+# Or, without axisartist, one can use secondary axes to add the additional
+# axes:
+
+fig, ax = plt.subplots()
+fig.subplots_adjust(right=0.85)
+
+ax.spines["top"].set_visible(False)
+ax.spines["right"].set_visible(False)
+
+ax1 = ax.secondary_xaxis(0)
+ax1.spines["bottom"].set_position(("data", 0))
+ax1.set_xlabel("Axis Zero")
+
+ax2 = ax.secondary_yaxis(1)
+ax2.spines["right"].set_position(("outward", 20))
+ax2.set_ylabel("Label Y2")
+
+ax.plot([-2, 3, 2])
+
```

Comment:
OK, this is nice.  Can this be added as an example in the main spines section as well?  Or moved there, with a link to it here?  Putting this second in this section makes it look like this is a deprecated way to do it, whereas I think the goal is for it to be the canonical way.

Reply:
I removed the additions to simple_axisline2 and simple_axisartist1 and just linked them https://matplotlib.org/devdocs/gallery/ticks_and_spines/spine_placement_demo.html#sphx-glr-gallery-ticks-and-spines-spine-placement-demo-py instead.

I think I'll leave the multi-axes example as it is right now because there's no completely equivalent example elsewhere and I don't want to bother about proper explanations for now, but we can always move it there later (e.g. when we kill SubplotZero...).